apps/wallet/src/components/CheckedInput.tsx

```diff
@@ -1,31 +1,44 @@
-import type { FC } from 'react';
-import { useState } from 'react';
+import type { FC, ReactNode } from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
 import { StyleSheet, Text, View } from 'react-native';
-import type { InputProps } from '@walless/gui';
 import { Input } from '@walless/gui';
 
 type Props = {
-	checkFunction: (value?: string) => string | undefined;
-} & InputProps;
-
-const CheckedInput: FC<Props> = ({ value, checkFunction, ...inputProps }) => {
-	const [errorText, setErrorText] = useState<string>();
-
-	const handlerBlur = () => {
-		setErrorText(checkFunction(value));
-	};
+	value?: string;
+	placeholder?: string;
+	keyboardType?: KeyboardTypeOptions;
+	errorText?: string;
+	suffix?: ReactNode;
+	onChangeText?: (value?: string) => void;
+	onBlur?: (value?: string) => void;
```

Comment:
`onBlur` should not have a param, just pass it like the original `onBlur`. It's like exposing `onBlur` prop from the component

Reply:
also, you should keep the props when wrapping a component if you don't override it